src/auth/local-auth.guard.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class LocalAuthGuard extends AuthGuard('local') {}
```

Comment:
```suggestion
export class JWTAuthGuard extends AuthGuard('jwt') {}
```

Phải dùng JwtStrategy chứ ko phải là local